scripts/modelbench/inductor_single_run.sh

```diff
@@ -51,11 +44,24 @@ if [[ "$DT" == "amp_fp16" ]]; then
 fi
 
 Flag_extra=""
-if [[ ${FREEZE} == "on" ]]; then
+cd /workspace/pytorch/benchmarks/dynamo
+if [[ $BACKEND == "aot_inductor" ]]; then
+    echo "Testing with aot_inductor."
```

Comment:
Those changes look a little bit complex, could we define a `${backend_extra}` to handle different backend?

Reply:
Submit a PR for this modification: https://github.com/chuanqi129/inductor-tools/pull/132 
1. `Backend_extra` is defined and replaced `Flag_extra`
2. Set `Freezing` by default for all backend

Reply:
Fixed by https://github.com/chuanqi129/inductor-tools/pull/132